cafebabel/templates/articles/drafts/edit.html

```diff
@@ -1,49 +1,71 @@
 {% extends "base.html" %}
 
+{% block body_class %}create-article-page{% endblock %}
 {% block body %}
 <form method=post action={{ url_for('drafts.edit', draft_id=article.id) }} enctype=multipart/form-data>
   <input type=hidden name=id value="{{ article.id }}">
-  <div class=text>
+  <div class=file>
+      <label for=image>Image</label>
+      {% if article.has_image %}
+          <img src={{ article_image_url(article) }} alt>
+          <div class=checkbox>
+            <label for=delete-image>Delete image</label>
+            <input type=checkbox name=delete-image id=delete-image value=1>
+          </div>
+      {% endif %}
+      <input type=file id=image name=image>
+  </div>
+  <h1 class='edit active'>
     <label for=title>Title</label>
     <input id=title name=title value="{{ article.title }}" required>
-  </div>
-  <div class=select>
-    <label for=language>Language
-      <select id=language name=language required>
-        {% for language in get_languages() %}
-        <option value={{language[0]}}{% if article.language == language[0] %} selected{% endif %}>{{ language[1] }}</option>
-        {% endfor %}
-      </select>
-    </label>
-  </div>
-  <div class=select>
-    <label for=category>Category
-      <select id=category name=category required>
-        {% for category in get_categories() %}
-        <option value={{ category }}{% if article.category == category %} selected{% endif %}>{{ category }}</option>
-        {% endfor %}
-      </select>
-    </label>
-  </div>
-  {% if current_user.has_role('editor') %}
-  <div class=select>
-    <label for=status>Status
-      <select id=status name=status required>
-        <option value=draft selected>Draft</option>
-        <option value=published>Published</option>
-      </select>
-    </label>
-  </div>
-  <div class=select>
-    <label for=author>Author
-      <select id=author name=author required>
-        {% for author in authors %}
-        <option value={{author.id}} {% if article.author == author %}selected{% endif %}>{{ author }}</option>
-        {% endfor %}
-      </select>
-    </label>
-  </div>
-  {% endif %}
+  </h1>
+    <div class=article-settings>
+      <div class=select>
+        <label for=language>
+          <select id=language name=language required>
+            <optgroup label=Language>
+              {% for language in get_languages() %}
+              <option value={{language[0]}}{% if article.language == language[0] %} selected{% endif %}>{{ language[1] }}</option>
+              {% endfor %}
+            </optgroup>
+          </select>
+        </label>
+      </div>
+      <div class=select>
+        <label for=category>
```

Comment:
My concern was that we do have a label tag, but no label content.
Suggestion was to insert a text that would be eventually removed from the screen with CSS `<label> <span>My non visible label</span>…`.

However I'm not aware whether or not `<optgroup label=xxx>` is accessible and enough for replacing a label? I can find documentation on regular label but not optgroup poorly documented and not so widely used.

In short: I don't have enough knowledge on this, so whatever you know/feel is best for the project 🥉

Reply:
haha ok